website/docs/d/cm_volume.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: volume"
+sidebar_current: "docs-rightscale-datasource-volume"
+description: |-
+  Defines a volume datasource to operate against.
+---
+
+# rightscale_volume
+
+Use this data source to get the ID of an existing volume for use in other resources.
+
+## Example Usage 1: Basic configuration of a volume data source
+
+```hcl
+data "rightscale_volume" "a_volume" {
+  cloud_href = "/api/clouds/1"
+
+  filter {
+    name = "my_volume"
+  }
+}
+
+output "volume name" {
+  value = "${data.rightscale_volume.crunis_volume.name}"
+}
+
+output "volume ID" {
+  value = "${data.rightscale_volume.crunis_volume.resource_uid}"
+}
+```
+
+## Argument Reference
```

Comment:
Your example includes 'cloud_href' but is not listed in the argument reference section.

Additionally, can you indicate which of these arguments are required vs. optional in parens so it matches the other datasource docs?